sharktank/sharktank/ops/default_impls.py

```diff
@@ -80,6 +81,20 @@ def elementwise_binary(operator, x, y):
     return operator(x, y)
 
 
+@elementwise.override(
```

Comment:
Can you ask a comment with some examples of what this overload matches?

Reply:
I added documentation and reworked the function to not use recursion.